hack/ci/windows.ps1

```diff
@@ -789,28 +789,22 @@ Try {
             $TestRunExitCode = $LastExitCode
             $ErrorActionPreference = "Stop"
 
-            # Saving for artifacts......
-            $unitTestsContPath="$ContainerNameForUnitTests`:c`:\gopath\src\github.com\docker\docker\bundles"
-            $JunitExpectedContFilePath = "$unitTestsContPath\junit-report-unit-tests.xml"
-            docker cp $JunitExpectedContFilePath "$TEMPORIG"
-            if (-not($LastExitCode -eq 0)) {
-                Throw "ERROR: Failed to docker cp the unit tests report ($JunitExpectedContFilePath) to $TEMPORIG"
-            }
-
-            if (Test-Path "$TEMPORIG\junit-report-unit-tests.xml") {
-                Write-Host -ForegroundColor Magenta "INFO: Unit tests results($TEMPORIG\junit-report-unit-tests.xml) exist. pwd=$pwd"
-            } else {
-                Write-Host -ForegroundColor Magenta "ERROR: Unit tests results($TEMPORIG\junit-report-unit-tests.xml) do not exist. pwd=$pwd"
-            }
-            
             # Saving where jenkins will take a look at.....
             $bundlesDir = "bundles"
```

Comment:
Probably for a follow-up; we should get rid of some of these variables (I never was a big fan of them when they were added), and just inline the values; I think that's more readable, and doesn't require looking up what the actual path is that's set in the variable

Reply:
Actually; if this is always "bundles" perhaps we should do it as part of this PR (if you agree with that change)

Reply:
Sure. I can hardcode those.